docs/running/running.rst

```diff
@@ -48,8 +46,8 @@ Run ``python HelloWorld.py --help`` to see a complete list of available options.
 For something beyond a "Hello, world!" example, refer to :ref:`runningDetail`.
 
 
-Running CWL workflows
----------------------
+CWL Quickstart
+--------------
 
```

Comment:
We need to add the CWL example which is in the google docs and move most of the cruft below to some more suited section (or just delete it)